app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -104,7 +104,9 @@ def link_to_field(name, value, label = nil, facet_hash = {})
     # @see #index_field_link params
     # @return [Date]
     def human_readable_date(options)
-      Date.parse(options[:value]).to_formatted_s(:standard)
```

Comment:
I'm not sure what it is, but was getting a type error converting array to string. I didn't expect it tot be an array, since I don't see that in Solr. Help appreciated.

![screen shot 2016-08-10 at 3 36 52 pm](https://cloud.githubusercontent.com/assets/4163828/17568149/88b61a0e-5f10-11e6-9b14-904cd18f5cf8.png)